src/node_process_methods.cc

```diff
@@ -490,9 +543,26 @@ static void InitializeProcessMethods(Local<Object> target,
   env->SetMethod(target, "reallyExit", ReallyExit);
   env->SetMethodNoSideEffect(target, "uptime", Uptime);
   env->SetMethod(target, "patchProcessObject", PatchProcessObject);
+
+  target
+      ->Set(env->context(),
+            FIXED_ONE_BYTE_STRING(env->isolate(), "hrtime"),
+            FastHrtime::New(env))
+      .ToChecked();
 }
 
 }  // namespace node
 
+namespace v8 {
+template <>
+class WrapperTraits<node::FastHrtime> {
+ public:
+  static const void* GetTypeInfo() {
```

Comment:
i'm not entirely sure what this is used for or if it is correct /cc @MayaLekova

Reply:
I don't think V8 currently makes use of it. It should be okay for now.

Reply:
It was supposed to help V8 differentiate between different embedder API types to perform type checks for the signature of the function. We've moved away from this design, as v8::Signature already provides the necessary info for the receiver and we have the checks implemented in TurboFan, so there's no need to introduce an additional burden for the embedder to provide the type information twice.
In short, the current implementation you added is perfectly valid.